.github/workflows/ci.yml

```diff
@@ -58,9 +58,48 @@ jobs:
       with:
         command: build
         args: --features webview --example webview_custom_protocol
-    # fails because it needs uikit, which is not
-    # building for me
-    #- uses: actions-rs/cargo@v1
-    #  with:
-    #    command: build
-    #    args: --example ios-beta
+    - uses: actions-rs/toolchain@v1
+      with:
+          toolchain: stable
+          override: true
+          target: x86_64-apple-ios
+    # Since it's all Objective-C message passing under the hood, we're
+    # really just looking for whether we've broken the iOS build. It is likely
+    # that more robust tests/checking infrastructure should exist for this side
+    # of things as the iOS portion gets iterated on.
+    #
+    # (e.g, this at the moment will not catch invalid selector calls, like if an appkit-specific
+    # selector is used for something on iOS)
+    - uses: actions-rs/cargo@v1
+      with:
+        command: build
+        args: --target x86_64-apple-ios --example ios-beta --no-default-features --features uikit,autolayout
+
+  ios:
+    name: Check that iOS tests pass via dinghy.
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions-rs/toolchain@v1
+      with:
+          toolchain: stable
+          override: true
+          target: x86_64-apple-ios
+    - uses: actions-rs/install@v0.1
+      with:
+        crate: cargo-dinghy
+        version: latest
+        use-tool-cache: true
```

Comment:
I'm tempted to take some from [`objc2`'s CI to cache and reduce CI times](https://github.com/madsmtm/objc2/blob/3cbef36db8f9f18de4e8be1f6ea69f04aa424c92/.github/workflows/ci.yml#L320-L322) but I'm trying not to expand scope too much.

Reply:
Eh, I wouldn't mind it here - up to you if you want it bad enough or want the hassle of PRing it, heh.

Reply:
Done. Adding a [cache-aware cargo-install](baptiste0928/cargo-install) reduced CI from ~11 minutes to ~6 minutes.